app/models/hangout.rb

```diff
@@ -0,0 +1,11 @@
+class Hangout < ApplicationRecord
+  belongs_to :user
+  belongs_to :hangout_chat
```

Comment:
Should this be has_one maybe? I believe hangout is the parent in this case but I could be wrong

Reply:
Yep you're right! This was the one I was super iffy about. Both ways should be a has_one right? As in Hangout has one hangout chat and hangout chat has one hangout?